docs/web-wallet.md

```diff
@@ -0,0 +1,252 @@
+- **Embedded Issuance**: The Web Wallet provides embedded issuance, allowing users to claim their credentials within the verification flow if they haven't done so before.
+
+:::note
+
+Currently, the tool supports embedded issuance for only two types of credentials:
+
+- Proof of Identity (refer to the schema [here](https://github.com/anima-protocol/claims-polygonid/blob/main/schemas/json-ld/poi-v1.json-ld))
+- Proof of Life (refer to the schema [here](https://github.com/anima-protocol/claims-polygonid/blob/main/schemas/json-ld/pol-v1.json-ld))
+
+The tool still supports the verification of other credentials, which can be issued on the Polygon ID Wallet App.
+:::
+
+- **Simplified Integration**: Integration into your application is effortless, reducing the development workload and ensuring swift implementation.
+- **On-Chain and Off-Chain Query Verification**: Supporting both on-chain and off-chain query verification, the Web Wallet offers flexibility to meet diverse verification needs.
+- **Consistency and Accuracy**: With a standardized interface, the tool promotes uniformity across users and organizations, minimizing errors.
+
+## Getting Started
+//add yt video
+
+You can experience a demo of the Web Wallet by visiting this [link](https://web-wallet-demo.internal-polygonid-prod.com/)
+
+## User Interface and Experience
+
+### Embedded Issuance and Verification Journey for End Users
+
+Let's walk through an example scenario: Suppose a user needs to validate their humanity to perform specific actions (e.g. registration or posting a review) within an application.
+
+1. **Initiating the Process**: The user clicks the designated button on the application, triggering redirection to the Web Wallet.
+2. **Connecting the Wallet**: Upon arrival, the user selects 'Connect Wallet' and connects their Ethereum wallet with the tool.
+3. **Creating Identity and Accessing Credentials**: Next, the user selects 'Sign Message' to create a derived identity wallet and retrieve the encrypted credentials associated with their Ethereum wallet via the cloud storage.
+4. **Claiming the Credential**: If the user hasn't previously claimed the credential required by the verifier’s request, they proceed by clicking 'Claim Credential' and complete the issuance process (a face scan in this scenario) to claim the credential via the embedded issuance.
+5. **Proof generation**: Once the user owns the credential, can select 'Verify', triggering the generation of the zk-proof which is then shared to the verifier application in case of off-chain verification and to the smart contract in case of on-chain verification
+6. **Verification Completion**: The user finally clicks the ‘Continue’ button, upon which they are redirected to the application where the verification can be completed and the application flow can continue.
+
+### End-User Journey of Verification of other credentials
+
+When proving custom authentication on an application:
```

Comment:
This "proving custom authentication" sounds weird to me. I'm not even sure what "proving a custom authentication" is. I'd consider changing this to:

> When verifying credentials lacking support for embedded issuance on an application:

In any case, I have problems with this whole section. It repeats the previous steps but explains the option “Verify from mobile”, which should be also explained in the previous section.

I would consider completely removing this section and explaining the option “Verify from mobile” in the previous section. Then, we can add an info box below explaining this custom credential scenario:

ℹ️ For credentials that do not support embedded issuance directly through the Web Wallet, applications can still facilitate a seamless verification process. Users who have previously issued credentials stored on their mobile devices can continue the verification from their mobile device. To do this, users can simply click the “Verify from Mobile” button in the Web Wallet interface. This action seamlessly integrates with the existing credentials on their device, allowing them to continue with the verification flow without interruption.

Reply:
Fixed